module_1/first-react-app/src/components/MovieSortBy.js

```diff
@@ -0,0 +1,35 @@
+import React, {  useEffect } from 'react';
+import '../css/MoviesSortBy.css';
+
+/**
+ * Requirements
+ * --------------
+ * This component will render a label "Sort by" and a select control to the right. Select should have the following options:
+ *  Release Date
+ *  Title
+ * 
+ * The component should take a property that specifies current selection. Additionally, it should take a callback property to handle selection changes. The callback should be called every time a user changes "Sort by" value. The new value should be passed in callback arguments.
+ * Use Storybook when implementing every individual component. Write a story for every component you create. It will help you to check the result and play around with some interactivity before you embed components into the app.
+ * 
+ */
+const MovieSortBy = ({ sortBy, onSortChange }) => {
+    
+    useEffect(() => {
+        window.M.AutoInit();
+    }, []);
+
+    return (
+        <>
+            <div className="input-field dropdown-container">
+                <span className="label" >Sort By : </span>
```

Comment:
`<span className="label">` - the space deleted

Reply:
Corrected as per suggestion